applications/StructuralMechanicsApplication/tests/test_nodal_damping.py

```diff
@@ -65,32 +65,36 @@ def _set_and_fill_buffer(self,mp,buffer_size,delta_time):
     def test_nodal_damping(self):
         current_model = KratosMultiphysics.Model()
         mp = current_model.CreateModelPart("sdof")
+        mp.ProcessInfo[KratosMultiphysics.DOMAIN_SIZE] = 3
 
         self._add_variables(mp)
 
-        #create node
+        # Create node
         node = mp.CreateNewNode(1,0.0,0.0,0.0)
         node.AddDof(KratosMultiphysics.DISPLACEMENT_X)
         node.AddDof(KratosMultiphysics.DISPLACEMENT_Y)
         node.AddDof(KratosMultiphysics.DISPLACEMENT_Z)
 
-        #add bcs and initial values
+        # Add bcs and initial values
         init_displacement = 0.1
         init_velocity = 0.0
         node.Fix(KratosMultiphysics.DISPLACEMENT_X)
         node.Fix(KratosMultiphysics.DISPLACEMENT_Z)
         node.SetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_Y,0,init_displacement)
 
-        #create element
-        element = mp.CreateNewElement("NodalConcentratedDampedElement3D1N", 1, [1], None)
+        # Create element
+        prop_id = 0
+        properties = mp.Properties[prop_id]
         mass = 1.0
         stiffness = 10.0
         damping = 1.0
-        element.SetValue(KratosMultiphysics.NODAL_MASS,mass)
-        element.SetValue(StructuralMechanicsApplication.NODAL_DISPLACEMENT_STIFFNESS,[0,stiffness,0])
-        element.SetValue(StructuralMechanicsApplication.NODAL_DAMPING_RATIO,[0,damping,0])
+        properties.SetValue(StructuralMechanicsApplication.CONSIDER_RAYLEIGH_DAMPING, False)
```

Comment:
try not to set any of this to ensure that it also works out of the box.

Reply:
This is an standalone test, there are other tests using it directly on mdpa. As this PR is retro-compatible I don't need to modify that